torch/optim/lr_scheduler.py

```diff
@@ -1,5 +1,6 @@
 from bisect import bisect_right
 from .optimizer import Optimizer
+from .. import load, save
```

Comment:
nit: I'd change that to `import torch` and use `torch.load` and `torch.save` to make it clear that these are not `load`, `save` methods of any class (that's what I thought when I saw them in the code)